frontend/src/screens/ConnectAlbyAccount.tsx

```diff
@@ -47,10 +47,19 @@ export function ConnectAlbyAccount({ connectUrl }: ConnectAlbyAccountProps) {
               </CardDescription>
             </CardHeader>
           </Card>
+          <Card className="w-full">
```

Comment:
This will only work for LDK

Reply:
Also, we do not know at startup if they will choose LDK, because this screen is shown before picking the backend type.

Reply:
I don't know how to do this without a*

* Encrypted backups are only available for the Alby Hub embedded node. Read more. (guide link)

Thoughts? @bumi @reneaaron

Reply:
I was actually thinking of adding some "security" page. Which describes all the details from the importance of the password, the seed, and also these backup options. 
So we can link there and people have to acknowledge it similar to a ToS.

Reply:
@bumi that sounds great. I think we also want to advertise backups as a feature worth having an Alby Account though, right?

Reply:
I will move your idea to a new issue.

Reply:
yeah, I think this change is good. I don't think we need the disclaimer here when we talk more about security and backups in detail and link to that.